CHANGELOG.md

```diff
@@ -1,14 +1,14 @@
-## 0.11.4-dev2
+## 0.11.4-dev3
 
 ### Enhancements
 
 * **Refactor pdfminer code.** The pdfminer code is moved from `unstructured-inference` to `unstructured`.
+* **Improve handling of auth data for fsspec connectors** Leverage an extension od the dataclass paradigm to support a `sensitive` annotation for fields related to auth (i.e. passwords, tokens). Refactor all fsspec connectors to use explicit access configs rather than a generic dictionary.
```

Comment:
Picky of me but can you fix the od to of. I stopped for a second to decypher od and then realized its of. Since it will probably be read by many.